include/glow/Runtime/Provisioner/Provisioner.h

```diff
@@ -67,8 +67,12 @@ class Provisioner final {
   std::vector<DeviceManager *> devices_;
 
   /// Helper function to cleanup a provision call. On failure free the
-  /// compiledFunctions that were created.
-  void cleanupProvision(llvm::ArrayRef<std::string> names, bool failure = true);
+  /// compiledFunctions that were created and remove and networks already added
+  /// to devices.
```

Comment:
nit: Reference the parameters in doxygen? E.g. you we remove all networks already added to devices, but this is actually based on whatever is found in `\p deviceCleanup`.